bittensor/core/extrinsics/asyncex/registration.py

```
@@ -135,90 +83,119 @@ async def burned_register_extrinsic(
 
     logging.debug(":satellite: [magenta]Recycling TAO for Registration...[/magenta]")
 
-    success, err_msg = await _do_burned_register(
-        subtensor=subtensor,
-        netuid=netuid,
+    # create extrinsic call
+    call = await subtensor.substrate.compose_call(
+        call_module="SubtensorModule",
+        call_function="burned_register",
+        call_params={
+            "netuid": netuid,
+            "hotkey": wallet.hotkey.ss58_address,
+        },
+    )
+    fee = await get_extrinsic_fee(
+        subtensor=subtensor, call=call, keypair=wallet.coldkeypub
+    )
+    logging.info(
+        f"The registration fee for SN #[blue]{netuid}[/blue] is [blue]{fee}[/blue]."
+    )
+    success, message = await subtensor.sign_and_send_extrinsic(
+        call=call,
         wallet=wallet,
         wait_for_inclusion=wait_for_inclusion,
         wait_for_finalization=wait_for_finalization,
         period=period,
+        raise_error=raise_error,
     )
 
     if not success:
-        logging.error(f":cross_mark: [red]Failed error:[/red] {err_msg}")
+        logging.error(f":cross_mark: [red]Failed error:[/red] {message}")
         await asyncio.sleep(0.5)
         return False
+
+    # TODO: It is worth deleting everything below and simply returning the result without additional verification. This
+    #  should be the responsibility of the user. We will also reduce the number of calls to the chain.
```

Technically yes we should return the result. Remnants of old design. 

From the other perspective, its useful to know additional stats in subnets with uid pressure

---

I thought about it. Maybe it's worth implementing an additional flag, something like `extended_info`. If it's passed to True, then calculate. If not, then just silently skip. But I'm still thinking about it. Let's leave it as is in this PR, I'm not deleting anything anyway. I'll come back to this at the TODO processing step.
Does it make sense?

---

Sounds good!